packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -208,10 +208,20 @@ class IndividualAction extends Component {
     (disabled, item, userData) =>
       isFunction(disabled) ? disabled(item, userData) : disabled
   )
+  _getHandlerParam = createSelector(
```

Comment:
You shouldn't need these selectors, actually. Just change the handlers as you need it.